packages/client/hmi-client/src/services/model-configurations.ts

```diff
@@ -197,22 +198,40 @@ export function getParameter(
 	return config.configuration.semantics?.ode.parameters?.find((param) => param.id === parameterId);
 }
 
-export function setDistribution(
-	config: ModelConfiguration,
-	parameterId: string,
-	distribution: ModelDistribution
-): void {
+export function getParameterName(config: ModelConfiguration, parameterId: string): string {
 	const parameter = getParameter(config, parameterId);
-	if (parameter) {
-		parameter.distribution = distribution;
-	}
+	return parameter?.name ?? '';
 }
 
-export function removeDistribution(config: ModelConfiguration, parameterId: string): void {
+export function getParameterDescription(config: ModelConfiguration, parameterId: string): string {
 	const parameter = getParameter(config, parameterId);
-	if (parameter?.distribution) {
-		delete parameter.distribution;
+	return parameter?.description ?? '';
+}
+
+export function getParameterUnit(config: ModelConfiguration, parameterId: string): string {
+	const parameter = getParameter(config, parameterId);
+	return parameter?.units?.expression ?? '';
+}
+
+export function getParameterDistribution(
+	config: ModelConfiguration,
+	parameterId: string
+): ModelDistribution {
+	const parameter = getParameter(config, parameterId);
+
+	if (!parameter?.distribution) {
+		return {
+			type: DistributionType.Constant,
+			parameters: {
+				value: parameter?.value
+			}
+		};
 	}
+
+	if (parameter.distribution.type === 'Uniform1')
+		parameter.distribution.type = DistributionType.Uniform;
+
+	return parameter.distribution;
```

Comment:
I think we need a few minor tweaks here.

In one instance, you are returning a new copy (constant-distribution). 
In another you are returning a modified reference of the config object.

So there are two problems here:
- It is inconsistent
- Your getter changes the underlying object

I am guess you probably want to do something like this?
```
const r = clone(parameter.distribution);
if (r.distribution.type === ....) r.distribution.type = ....
return r
```

Reply:
👍 it does not return a reference now